components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController+Testing.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBottomDrawerContainerViewController.h"
+
+extern NSString *_Nonnull const kMDCBottomDrawerScrollViewAccessibilityIdentifier;
```

Comment:
I feel it may be reasonable for us to expose this accessibility identifier as a public API (changes to it should be considered breaking changes for any downstream clients); it looks like you may feel otherwise though. Can you expand on the reasoning for making it private?

Reply:
Doing so would also have the benefit of simplifying this PR.

Reply:
Sure. A client has asked us to provide an a11y identifier to the internal scrollview of the drawer that is a subview of our private implementation class MDCBottomDrawerContainerViewController. Therefore I thought it makes sense to leave it under private/ and expose it under testonly=1 in the BUILD file. Do you feel like we should have this header file be imported under the umbrella header and moved under the main src/ folder?

Reply:
*nods* that reasoning seems fair to me.